drivers/note/Kconfig

```diff
@@ -36,6 +36,19 @@ config DRIVERS_NOTECTL
 		If this option is selected, the instrumentation filter control device
 		/dev/notectl is provided.
 
+config NOTE_GET_PERF_TIME
+	bool "Note get time config"
+	default y
+	---help---
+		If this option is selected, then note get time from perf time
```

Comment:
Please improve the "---help---", what is "perf time"? What difference to "clock time", etc. These are question that user need to know before enabling this option